core/storage/question/gae_models.py

```diff
@@ -600,14 +631,19 @@ def get_deletion_policy():
         """
         return base_models.DELETION_POLICY.LOCALLY_PSEUDONYMIZE
 
+    @staticmethod
+    def get_export_policy():
+        """Model does not contain user data."""
```

Comment:
Please fix this docstring. (See comments above.)

Reply:
Updated.